tools/travis/script.sh

```diff
@@ -22,6 +27,8 @@ section "Tests.examples"
 # Run example applications
 echo Build or run examples
 pip install --retries 3 -q -r ./requirements/docs.txt
+pip list
+tools/build_versions.py
```

Comment:
This looks redundant, doesn't it?

Reply:
I think both are useful. `build_versions` gives us a quick at a glance look of what we installed. `pip list` is very verbose. Maybe putting it in 

```
 section "examples.install.dependencies"
pip install --retries 3 -q -r ./requirements/docs.txt
pip list
tools/build_versions.py
 section_end "examples.install.dependencies"
```

would help clarify things?